src/injected/visualization/svg-drawer.ts

```diff
@@ -265,11 +274,44 @@ export class SVGDrawer extends BaseDrawer {
         );
     }
 
-    private shouldBreakGraph(curElementIndex: number): boolean {
+    private createFocusIndicatorForFailure(item: TabbedItem): FocusIndicator {
+        const centerPosition: Point = this.centerPositionCalculator.getElementCenterPosition(
+            item.element,
+        );
+
+        if (centerPosition == null) {
+            return;
+        }
+
+        const drawerConfig: SVGDrawerConfiguration = this.formatter.getDrawerConfiguration(
+            item.element,
+            null,
+        ) as SVGDrawerConfiguration;
+
+        const circleConfiguration =
+            item.itemType === TabbedItemType.ErroredItem
+                ? drawerConfig.erroredCircle
+                : drawerConfig.missingCircle;
+
+        const newCircle = this.svgShapeFactory.createCircle(centerPosition, circleConfiguration);
+        const newLabel = this.svgShapeFactory.createTabIndexLabel(
+            centerPosition,
+            drawerConfig.tabIndexLabel,
```

Comment:
This doesn't necessarily need to be done in this PR, but eventually we will want to use a different config for error labels since we will want the text on them to be red. See the erroredTabIndexLabel config I added here: https://github.com/microsoft/accessibility-insights-web/pull/5199/files#diff-00053e82d4b379e30958fd1ca42842a8fadbd20d903faaa76bc26e1397656017R66

Reply:
Good catch! I'll look into it!

Reply:
Added!